lms/js_config_types.py

```diff
@@ -17,38 +23,32 @@ class APICourse(TypedDict):
     title: str
 
 
-class APICourses(TypedDict):
-    courses: list[APICourse]
-
-
-class APIStudentStats(TypedDict):
-    display_name: str
-    annotations: int
-    replies: int
-    last_activity: str | None
-
+class APIStudent(TypedDict):
+    id: int
```

Comment:
Is this ID an `int`? I can see values like `d30bf96b4bd3d91fd48582e9c31f41941953bc30` or `e4ca30ee27eda1169d00b83f2a86e3494ffd9b12`.

Reply:
Yes, added that on a new commit now that this is a "shared branch".